.gitignore

```diff
@@ -334,3 +334,4 @@ ASALocalRun/
 
 .vscode/**/*.*
 DebugPlatforms.props
+.DS_Store
```

Comment:
FYI: This is an important addition. Git evidently picks up these temporary OS files in each directory on macOS, so I've ignored them.